{
  "id": "MAL-2024-5708",
  "summary": "Malicious code in pyfontinglib (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "aliases": [
    "SNYK-PYTHON-PYFONTINGLIB-5932223"
  ],
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:40:02Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "sha256": "4193e944604446d7313dccc4e939d55d2d10b2b27e5612a80be921fa04283ac6",
        "import_time": "2024-06-28T02:50:06.562795461Z",
        "versions": [
          "1.0.0"
        ],
        "id": "RLMA-2024-04492",
        "source": "reversing-labs",
        "modified_time": "2024-06-25T13:40:02Z"
      },
      {
        "sha256": "04413cc6f40ce29916e6ea8325b30a547836e48cec1dc2ca1a3ca8b89f0b25ee",
        "import_time": "2024-10-24T00:59:51.146780557Z",
        "id": "RLUA-2024-08930",
        "source": "reversing-labs",
        "modified_time": "2024-10-16T14:47:57Z"
      }
    ]
  },
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PYFONTINGLIB-5932223"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "pyfontinglib",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/pyfontinglib"
      },
      "versions": [
        "1.0.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/pyfontinglib/MAL-2024-5708.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}